mlrun/api/api/endpoints/schedules.py

```diff
@@ -28,6 +28,24 @@ def create_schedule(
     return Response(status_code=HTTPStatus.CREATED.value)
 
 
+@router.put("/projects/{project}/schedules")
+def update_schedule(
+    project: str,
+    name: str,
+    schedule: schemas.ScheduleUpdate,
+    db_session: Session = Depends(deps.get_db_session),
+):
+    get_scheduler().update_schedule(
+        db_session,
+        project,
+        name,
+        schedule.scheduled_object,
+        schedule.cron_trigger,
+        labels={label.name: label.value for label in schedule.labels},
+    )
+    return Response(status_code=HTTPStatus.ACCEPTED.value)
```

Comment:
I think this should be either 200 or 204 (https://stackoverflow.com/questions/2342579/http-status-code-for-update-and-delete#:~:text=For%20a%20PUT%20request%3A%20HTTP,resource%20was%20marked%20for%20deletion%22.)